package.json

```diff
@@ -17,13 +17,14 @@
     "@tailwindcss/container-queries": "^0.1.1",
     "@tailwindcss/typography": "^0.5.2",
     "@tanstack/react-query": "^5.0.0",
+    "@uidotdev/usehooks": "^2.4.1",
     "axios": "^1.1.3",
     "critters": "^0.0.20",
     "decanter": "^7.0.0-rc.2",
     "drupal-jsonapi-params": "^2.0.0",
     "html-react-parser": "^5.0.6",
     "jsona": "^1.11.0",
-    "next": "^14.0.1",
+    "next": "^13.0",
```

Comment:
What is not working in Next 14 that you had to roll back?

Reply:
The route interception kept throwing errors. I debugged it for a few hours and then this seemed to fix it. Not sure exactly why that solved the issue